static/js/entry/zendesk_widget.js

```diff
@@ -116,6 +118,46 @@ const zendeskCallbacks = {
         }, 100);
       };
     }
+  },
+  ticketSubmissionFormLoaded: () => {
+    const iframe = document.querySelector("iframe.zEWidget-ticketSubmissionForm");
+
+    const fieldSelector = name => (
+      `input[name="${name}"], select[name="${name}"]`
+    );
+    const iframeQuerySelector = query => (
+      iframe.contentDocument.querySelector(query)
+    );
+    const fieldElement = R.compose(
+      iframeQuerySelector, fieldSelector
+    );
+
+    // Zendesk uses the ID 24690866 to refer to the MicroMasters program selector
+    const programFieldName = '24690866';
+    const fieldVisibility = {
+      'name':  !(SETTINGS.user && SETTINGS.user.first_name
+                 && SETTINGS.user.last_name),
+      'email': !(SETTINGS.user && SETTINGS.user.email),
+    };
+    fieldVisibility[programFieldName] = !(
+      SETTINGS.program && SETTINGS.program.slug
+    );
```

Comment:
There is!

```js
const myKey = 'a key';

const obj = {
  otherKey: 'other value',
  [myKey]: 'my value',
};
```

the syntax feature is called a computed property name, there's some info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer